src/ui/profile_preferences.ui

```diff
@@ -55,6 +55,19 @@
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_19">
```

Comment:
We're not gonna have space for all the tabs in the dialog because there's a tab for the Editor coming as well, and that filled up the space to 100%... 

GIven how IRC is not gonna gain that many more options, how about we move it into `Special Options` tab?